extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -31,6 +31,17 @@ import {toggle} from '../../../src/style';
 
 const TAG = 'amp-consent-ui';
 
+// Classes for consent UI
+const consentUiClasses = {
+  fullscreen: 'i-amphtml-consent-ui-fullscreen',
```

Comment:
Why we need an object? Also I don't know if we are planning to move to JsonObject, I'm waiting for Erwin's response on this.

Reply:
I moved this into a class, per @lannka 's suggestion of moving the classes into constants so that we have one place to change it.

I also felt like `consentUiClasses.fullscreen`, would make more sense than `CONSENT_UI_CLASSES_FULLSCREEN`, and mantaining a bunch of individual variables.

But yes, I can set this as JSONObject type 😄

Reply:
Bracket notiation with Strings